components/radio/index.css

```diff
@@ -144,6 +146,11 @@ governing permissions and limitations under the License.
 
     /* high contrast focus ring */
      --highcontrast-radio-focus-ring-color: ButtonText;
+
+    --highcontrast-radio-button-border-color-default: ButtonText;
+    --highcontrast-radio-button-border-color-hover: ButtonText;
+    --highcontrast-radio-button-border-color-down: ButtonText;
+    --highcontrast-radio-button-border-color-focus: ButtonText;
```

Comment:
New custom properties with naming that is consistent with their `--spectrum-radio-button-*` counterparts.

Also, I _think_ `ButtonText` is what you're wanting here. That might need to change for the `down` and `focus` states, though.